monkey/monkey_island/cc/ui/src/styles/pages/report/ReportPage.scss

```diff
@@ -72,16 +72,25 @@ div.report-wrapper {
   padding-bottom: 20px;
 }
 
-div.report-wrapper .nav-tabs > .nav-item > a:hover:not(.active),  .nav-tabs > .nav-item > a:focus:not(.active){
-    text-decoration: none;
-    background-color: $light-gray;
+div.report-wrapper .nav-tabs > .nav-item > a:hover:not(.active), .nav-tabs > .nav-item > a:focus:not(.active) {
+  text-decoration: none;
+  background-color: $light-gray;
 }
 
 ul.cross-segment-issues {
-    list-style-type: none;
-    padding: 0px;
-    margin: 0px;
+  list-style-type: none;
+  padding: 0px;
+  margin: 0px;
 }
+
 span.cross-segment-service {
-    text-transform: uppercase;
+  text-transform: uppercase;
+}
+
+.report-page li a.btn,.security-report-link {
+  position: relative;
+  font-size: 1em;
+  padding: 0 5px;
+  line-height: 1em;
+  top: -3px;
 }
```

Comment:
@VakarisZ why do this and not just make "Restore your domain controller's password manually." a hyperlink?

Reply:
It is a hyperlink: https://react-bootstrap.github.io/components/buttons/